docs/zh/week12/12-2.md

```diff
@@ -0,0 +1,326 @@
+---
+lang-ref: ch.12-2
+title: Decoding Language Models
+lecturer: Mike Lewis
+authors: Trevor Mitchell, Andrii Dobroshynskyi, Shreyas Chandrakaladharan, Ben Wolfson
+date: 20 Apr 2020
+---
```

Comment:
`lang: zh`, `translator:` and `translation-date:` are missing.

Reply:
![image](https://user-images.githubusercontent.com/2119355/88357811-92a05600-cd3a-11ea-837a-a14af57a67c2.png)

I'm fixing this myself. Please, read the comments next time 😛